sources/packages/forms/src/form-definitions/supportingusersdashboard.json

```diff
@@ -167,265 +175,530 @@
+                          "prefix": "",
+                          "suffix": "",
+                          "multiple": false,
+                          "defaultValue": null,
+                          "protected": false,
+                          "unique": false,
+                          "persistent": false,
+                          "clearOnHide": true,
+                          "refreshOn": "",
+                          "redrawOn": "",
+                          "dataGridLabel": false,
+                          "labelPosition": "top",
+                          "description": "",
+                          "errorLabel": "",
+                          "tooltip": "",
+                          "hideLabel": false,
+                          "tabindex": "",
+                          "disabled": false,
+                          "autofocus": false,
+                          "dbIndex": false,
+                          "customDefaultValue": "",
+                          "calculateValue": "",
+                          "calculateServer": false,
+                          "widget": null,
+                          "validateOn": "change",
+                          "validate": {
+                            "required": false,
+                            "custom": "",
+                            "customPrivate": false,
+                            "strictDateValidation": false,
+                            "multiple": false,
+                            "unique": false
+                          },
+                          "allowCalculateOverride": false,
+                          "encrypted": false,
+                          "showCharCount": false,
+                          "showWordCount": false,
+                          "allowMultipleMasks": false,
+                          "addons": [],
+                          "id": "enzfeyr"
+                        },
+                        {
+                          "label": "Start submission",
+                          "labelWidth": "",
+                          "labelMargin": "",
+                          "action": "custom",
+                          "showValidations": false,
+                          "theme": "primary",
+                          "size": "md",
+                          "block": false,
+                          "leftIcon": "",
+                          "rightIcon": "",
+                          "shortcut": "",
+                          "description": "",
+                          "tooltip": "",
+                          "customClass": "",
+                          "tabindex": "",
+                          "disableOnInvalid": false,
+                          "hidden": false,
+                          "autofocus": false,
+                          "disabled": false,
+                          "tableView": false,
+                          "modalEdit": false,
+                          "key": "startParentInformationRequest1",
```

Comment:
maybe we can avoid keys ending with numbers like 1.

Reply:
yes, that came from the older version. I'll change that. Thanks!

Reply:
Changed keys in the form.